src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      width: "100%",
+      marginTop: "16px",
```

Comment:
I'm so sorry. Please remove margin-top because it will redundant with padding T^T.